pkg/process/checks/process.go

```diff
@@ -113,8 +115,10 @@ type ProcessCheck struct {
 
 	lookupIdProbe *LookupIdProbe
 
-	extractors         []metadata.Extractor
-	workloadMetaServer *workloadmeta.GRPCServer
+	extractors []metadata.Extractor
+
+	workloadmetaExtractor *workloadmeta.WorkloadMetaExtractor
```

Comment:
Nit: rename to `workloadMetaExtractor` to be consistent with `workloadMetaServer`

Reply:
Fixed!